### ISSUE TITLE
Probem typo

### ISSUE THREAD

**Issue by racke** [sympa-community/sympa#115]
```
./default/mail_tt2/user_notification.tt2:52:Please contact the listmaster (%2@%3) to fix this probem before re-posting your message.[% END %]
./po/sympa/sympa.pot:11327:"Please contact the listmaster (%2@%3) to fix this probem before re-posting your message."
```

**Comment by ikedas** [comment]
Also, "You message to list %1 could not be delivered." in the same entry would be "You*r* ...".

**Comment by ikedas** [comment]
Reported typos and corresponding catalog entries were [fixed](https://github.com/sympa-community/sympa/commit/1e2e0941fd771a702b7d04f2166bbe59d90ca6cd#diff-e18b607efb46e4cdee50fb21b43a2e07) (I wrote a script to reflect changes on en_US.po to the sources).

I suppose, to catch complaint about typos, this issue would be better to be left open.

**Comment by qosobrin** [comment]
#: src/lib/Sympa/ConfDef.pm:449
msgid "The number of seconds a bulk worker will remain running without processing a message before it spontaneously exists."

should be 

msgid "The number of seconds a bulk worker will remain running without processing a message before it spontaneously exits." 

or even 

msgid "The number of seconds a bulk worker will remain running without processing a message before it spontaneously quits."

**Comment by qosobrin** [comment]
#: src/lib/Sympa/ConfDef.pm:437
msgid "The minimum number of packets before bulk daemon forks the new worker to increase sending rate."

should be 

msgid "The minimum number of packets before bulk daemon forks a new worker to increase sending rate."

**Comment by ikedas** [comment]
@qosobrin, thanks for reporting typo.

If possible, please add your fix to english catalog:
  - On [English page of translation site](https://translate.sympa.org/en_US/), add correct string as a "translation" of incorrect msgid.

Your change on the site above will be reflected to source distribution in the future release.

**Comment by qosobrin** [comment]
Done. I could only "Suggest" my commits in the translation site; it didn't allow me to "Send"

**Comment by qosobrin** [comment]
By the way, what do you prefer me to do with next strings? Should I publish them here and commit the "translation" later? Or should I commit the translation directly?

**Comment by qosobrin** [comment]
I have a doubt about this string:

#: src/lib/Sympa/ConfDef.pm:694
msgid "Remove subscribers bouncing remind message"

Shouldn't it be "Remove bouncing subscribers remind message"? I read the source and I can not understand the context it is being used.

**Comment by ikedas** [comment]
> By the way, what do you prefer me to do with next strings? Should I publish them here and commit the "translation" later? Or should I commit the translation directly?

Please do how you feel reliable. Eventually, only committed translations will be adopted.

> I have a doubt about this string:
> 
> #: src/lib/Sympa/ConfDef.pm:694
> msgid "Remove subscribers bouncing remind message"
> 
> Shouldn't it be "Remove bouncing subscribers remind message"? I read the source and I can not understand the context it is being used.

More exactly, it means "If a subscription reminder (a message sent by `remind` operation) bound for an address of a subscriber bounces, remove that subscriber".

**Comment by racke** [comment]
Maybe "Remove subscribers with bounces on reminders." ?

The original text is indeed utterly confusing.

**Comment by qosobrin** [comment]
I have a doubt about this string:

#: src/lib/Sympa/ListDef.pm:950
msgid ""
"Each bouncing user have a score (from 0 to 100).\n"
"This parameter defines a lower limit for each category of bouncing users.For example, level 1 begins from 45 to level_2_treshold."

Is it correct the name of the variable it is referenced at the end of the string (level_2_treshold)? Should it be level_2_threshold instead?

**Comment by ikedas** [comment]
There are no parameter like `level_2_th`?`reshold`.  It would be `bouncers_level2.rate` or "*‘threshold’ of ‘Management of bouncers, 2nd level’*".  (Former notation uses parameter names visible only by listmasters).

**Comment by pkissman** [comment]
This is the above referenced closed issue #245 moved here.

A typo has snuck into a bunch of the po files. In en_US, see line 6292

msgid "**Dou** you really want to add %1?"

**Comment by qosobrin** [comment]
@pkissman The typo is already corrected in the [English catalog of the translation site](https://translate.sympa.org/en_US/translate/#search=Dou%20you%20really%20want%20to%20add%20%251&sfields=source,target) and available in [Sympa 6.2.30](https://github.com/sympa-community/sympa/blob/master/po/sympa/en_US.po)

Meanwhile you can correct it in your installation modifying the .mo file of your language. Remember that you will have to:

1. Convert the .mo file to a .po using the standard GNU Gettext tools.
2. Edit the .po file and change the msgstr value, not the msgid.
3. Convert back the .po file to a .mo

Regards.

**Comment by ikedas** [comment]
No more typos seem to be found.  Close.
